app/repositories/onlinetesting/Phase3TestRepository.scala

```diff
@@ -57,7 +57,9 @@ class Phase3TestMongoRepository(dateTime: DateTimeFactory)(implicit mongo: () =>
   override implicit val bsonHandler: BSONHandler[BSONDocument, Phase3TestGroup] = Phase3TestGroup.bsonHandler
 
   override def nextApplicationsReadyForOnlineTesting: Future[List[OnlineTestApplication]] = {
-    val query = BSONDocument("applicationStatus" -> ApplicationStatus.PHASE2_TESTS_PASSED)
+    val query = BSONDocument("applicationStatus" -> ApplicationStatus.PHASE2_TESTS_PASSED,
+      s"progress-status.${PHASE2_TESTS_COMPLETED}" -> true
```

Comment:
How is the user getting in a situation that they are in ApplicationStatus.PHASE2_TESTS_PASSED without a progress status of PHASE2_TESTS_COMPLETED?